charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team acts as a liaison between labs who want to contribute data to the Human Cell Atlas (HCA) and the Data Coordination Platform (DCP). This team works with data contributing labs, to understand each lab's data and ensure the experiments are well-represented in the platform and to support the lab writing data to the platform, working with them to resolve any issues in the process.
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who supports to data contributors wanting to upload data into the DCP
```

Comment:
Bit redundant. How about:

_- the Ingest Support team member who assists labs contributing data to the HCA._